src/main/java/nl/erwinvaneyk/core/logging/InfluxLogger.java

```diff
@@ -0,0 +1,115 @@
+package nl.erwinvaneyk.core.logging;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import nl.erwinvaneyk.communication.Message;
+import nl.erwinvaneyk.core.Address;
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.influxdb.dto.Database;
+import org.influxdb.dto.Serie;
+import retrofit.RetrofitError;
+
+@Slf4j
+public class InfluxLogger implements RawLogger {
```

Comment:
The point of available is that we don't want to care about the logging whether it is online or not. isAvailable does a complete ping-request to check if it is online. If it is not the available is set to false. The variable is stored because we don't want to do a ping-request everytime we try to log a message, which would cause lots of redundant messages to InfluxDB